rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.5.0"]
```

```suggestion
ecs_version = ["1.6.0"]
```

can accept this change once #220 merges